src/cpp/session/modules/SessionFiles.R

```diff
@@ -174,3 +174,19 @@ for (binding in bindings)
 {
    .Call("rs_readLines", path.expand(filePath))
 })
+
+.rs.addJsonRpcHandler("get_issue_url", function(id)
+{
+   project <- .rs.getProjectDirectory()
+   if (is.null(project)) {
+      return(.rs.scalar(""))
+   }
+   
+   BugReports <- as.character(read.dcf(file.path(project, "DESCRIPTION"), fields = "BugReports"))
+   if (is.na(BugReports)) {
+      return(.rs.scalar(""))
+   }
+
+   url <- paste0(BugReports, "/", sub("#", "", id))
```

Comment:
Currently: 
 - if BugReports is a GitHub url, then append the id
 - if this is any other url, just use that (no appending)
 - give up

It would be nice to allow issues from other systems (gitlab, ...) but I don't know how to do the mapping.